micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jetty/JettyStatisticsMetrics.java

```diff
@@ -0,0 +1,107 @@
+        bindGauge(reg,"jetty.dispatched.active", "Number of dispatches currently active", StatisticsHandler::getDispatchedActive);
+        bindGauge(reg,"jetty.dispatched.active.max", "Maximum number of active dispatches being handled", StatisticsHandler::getDispatchedActiveMax);
+        bindGauge(reg,"jetty.dispatched.time.max", "Maximum time spent in dispatch handling", StatisticsHandler::getDispatchedTimeMax);
+        bindGauge(reg,"jetty.async.requests.waiting", "Currently waiting async requests", StatisticsHandler::getAsyncRequestsWaiting);
+        bindGauge(reg,"jetty.async.requests.waiting.max", "Maximum number of waiting async requests", StatisticsHandler::getAsyncRequestsWaitingMax);
+
+        bindTimeGauge(reg,"jetty.request.time.max.seconds", "Maximum time spent handling requests", StatisticsHandler::getRequestTimeMax);
+        bindTimeGauge(reg,"jetty.stats.seconds", "Time in seconds stats have been collected for", StatisticsHandler::getStatsOnMs);
+
+        bindStatusCounters(reg);
+    }
+
+    private void bindStatusCounters(MeterRegistry reg) {
+        buildStatusCounter(reg,"1xx", StatisticsHandler::getResponses1xx);
+        buildStatusCounter(reg,"2xx", StatisticsHandler::getResponses2xx);
+        buildStatusCounter(reg,"3xx", StatisticsHandler::getResponses3xx);
+        buildStatusCounter(reg,"4xx", StatisticsHandler::getResponses4xx);
+        buildStatusCounter(reg,"5xx", StatisticsHandler::getResponses5xx);
+    }
+
+    private void bindGauge(MeterRegistry reg, String name, String desc, ToDoubleFunction<StatisticsHandler> func) {
+        Gauge.builder(name, statisticsHandler, func)
+            .description(desc)
+            .register(reg);
+    }
+
+    private void bindTimer(MeterRegistry reg, String name, String desc, ToLongFunction<StatisticsHandler> countFunc, ToDoubleFunction<StatisticsHandler> consumer) {
+        FunctionTimer.builder(name, statisticsHandler, countFunc, consumer, TimeUnit.MILLISECONDS)
+            .description(desc)
+            .register(reg);
+    }
+
+    private void bindTimeGauge(MeterRegistry reg, String name, String desc, ToDoubleFunction<StatisticsHandler> consumer) {
+        TimeGauge.builder(name, statisticsHandler, TimeUnit.MILLISECONDS, consumer)
+            .description(desc)
+            .register(reg);
+    }
+
+    private void bindCounter(MeterRegistry reg, String name, String desc, ToDoubleFunction<StatisticsHandler> consumer) {
+        FunctionCounter.builder(name, statisticsHandler, consumer)
+            .description(desc)
+            .register(reg);
+    }
+
+    private void buildStatusCounter(MeterRegistry reg, String status, ToDoubleFunction<StatisticsHandler> consumer) {
+        FunctionCounter.builder("jetty.responses", statisticsHandler, consumer)
+            .description("Number of requests with response status")
+            .tags("code", status)
```

Comment:
Can we name this tag "status" for consistency with `WebMvcTags`?

Reply:
Updated to `status`